examples_test.go

```diff
@@ -677,22 +678,22 @@ func ExampleTxn_Mutate_facets() {
 		NameOrigin: "Indonesia",
 		DType:      []string{"Person"},
 		Friends: []Person{
-			Person{
+			{
```

Comment:
this is implied (inherited) from the slice type?

Reply:
Yes